new_platforms/vsextension/ProjectWizard/ImportEnclaveCommand.cs

```diff
@@ -237,6 +237,7 @@ private async void Execute(object sender, EventArgs e)
                 {
                     // Get the path of specified file.
                     filePath = openFileDialog.FileName;
+                    WizardImplementation.EdlLocation = Path.GetDirectoryName(filePath);
```

Comment:
How does the wizard start after the Import Enclave command runs such that it consume this value? Is there a way to pass it to the wizard via a mechanism other than a static variable, perhaps `RunStarted`'s `customParams` argument (assuming one can control how the wizard's code runs)?

Reply:
Yes the code path to set the variable is executed before the code path where it is consumed.  There may be other ways to do it, I don't know.